apps/api/src/services/UserService.ts

```diff
@@ -78,12 +80,28 @@ export class UserService extends BaseService<User> {
     }
 
     async create(dto: User): Promise<User | undefined> {
+        const alreadyExists = await this._userRepository.findOne({
+            where: { email: dto.email }
+        });
```

Comment:
Can this not be done with setting the email field to unique? 

https://typeorm.io/#entities/column-options